### DISCUSSION TITLE
Important selector strategy allows for more configurations

### DISCUSSION THREAD

**Discussion by mirai027** [tailwindlabs/tailwindcss#11502] [Help]
When setting `important` to an ID selector `#app`, it generates code wrapped with `:is()` like `#app :is(.border-none)`.

However, the `:is()` selector cannot be used in browsers that do not support CSS selector `:is()`, such as versions older than Chrome 88.

Is it possible to add a configuration option to generate code like `#app .border-none` instead?

**Comment by wongjn** [comment]
No, there is no configuration option. As per #10835 that introduced the current behavior:

> The `:is(...)` pseudo-class has good browser support, with the most recent adopter being Safari in v14, which is almost three years old and two major versions in the past.

That being said, if you do have to support older browsers like Chrome 88, I offer my commiserations!
